CONTRIBUTING.md

```diff
@@ -644,6 +645,25 @@ Redoc generates HTML documentation using the InfluxDB `swagger.yml`.
 For more information about generating InfluxDB API documentation, see the
 [API Documentation README](https://github.com/influxdata/docs-v2/tree/master/api-docs#readme).
 
+## Canonical URLs
+Search engines use canonocial URLs to accurately rank pages with similar or identical content.
+The `canonical` HTML meta tag identifies which page should be used as the source of truth.
+
+By default, canonical URLs are automatically generated for each page in the InfluxData
+documentation using the latest version of the current product and the current path.
+
+Use the `canonical` frontmatter to override the auto-generated canonical URL.
```

Comment:
Is using the latest shortcode the only reason to override the auto-generated cononical URL? If yes, perhaps:
To use the `{{< latest >}}` shortcode, add the `canonical` frontmatter to override the auto-generated canonical URL.

Do we expand on why folks would use the latest shortcode somewhere?

Reply:
@kelseiv No, there are two use cases for this functionality:

1. If the canonical content is on a page that is different than the auto-generated canonical URL. For example, let's say we create the following page:

    `/influxdb/v2.0/this/page/`

    When we cut a v2.1 release, the autogenerated canonical URL for this page will be:

    `/influxdb/v2.1/this/page`

    BUT, what if we change the location of the page in v2.1 to `/influxdb/v2.1/that/page/` ? The canonical URL on `/influxdb/v2 will point to a page that doesn't exist or gets redirected. We can override it to point to the new page.

2. We can begin to pass the SEO strength of the 1.x docs to the v2 docs by pointing canonical links to v2 docs.